src/components/CreateNewProjectWizard/MainPane.js

```diff
@@ -29,29 +31,116 @@ type Props = {
   handleSubmit: () => Promise<any> | void,
 };
 
-class MainPane extends PureComponent<Props> {
+type State = {
+  gatsbyStarter: string, // Temporary value during selection in selection toast
+};
```

Comment:
hm it feels like this state lives in so many places... Couldn't this invoke `updateFieldValue` instead of being set locally?

Reply:
Yes, that's right - it lives in many places. But I'm not sure if I can simplify this. As the user can open the starter selection component and select a starter but then decides to cancel that selection so the previously entered starter is still available.

If I would use `updateFieldValue` that flow wouldn't work.